src/makeInfinityAdder.js

```diff
@@ -5,6 +5,31 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let total = 0;
+  const adder = (a) => {
+    total = a
+    if (a === undefined) {
```

Comment:
This line is unnecessary and disrupts the flow of the function. It should be removed to allow the initial call to `adder` to handle the addition correctly.